packages/bridge-controller/package.json

```diff
@@ -66,7 +66,7 @@
   },
   "devDependencies": {
     "@metamask/accounts-controller": "^32.0.1",
-    "@metamask/assets-controllers": "^73.0.2",
+    "@metamask/assets-controllers": "^73.1.0",
```

Comment:
Shouldn't we add an entry to the bridge-controller changelog when bumping a minor?

I've seen it done for other releases.

https://github.com/MetaMask/core/pull/6241/files

Reply:
It's a dev dependency bump, I think that's ok, we usually add entries for peer dep bumps (because they are breaking changes for consumers) and sometimes for "direct deps" (but that's up to the team that own the package)